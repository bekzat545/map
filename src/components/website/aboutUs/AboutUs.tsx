/* External dependencies */
import React from 'react';

export default function AboutUs() {
  return <div id="about-us">О проекте</div>;
}
