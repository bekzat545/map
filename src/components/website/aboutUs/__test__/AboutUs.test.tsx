/* External dependencies */
import React from 'react';
import { I18nextProvider } from 'gatsby-plugin-react-i18next';
import renderer from 'react-test-renderer';

/* Local dependencies */
import i18n from '../../../../../tests/utils/i18nForTest';
import AboutUs from '../AboutUs';

describe('AboutUs component', () => {
  function getRenderComponent() {
    return (
      <I18nextProvider i18n={i18n}>
        <AboutUs />
      </I18nextProvider>
    );
  }

  it('should match the snapshot', () => {
    const component = renderer.create(getRenderComponent());

    expect(component.toJSON()).toMatchSnapshot();
  });
});
