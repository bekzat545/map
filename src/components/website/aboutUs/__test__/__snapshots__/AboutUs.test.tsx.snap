// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs component should match the snapshot 1`] = `
<div
  id="about-us"
>
  О проекте
</div>
`;
